practice.tf

```diff
@@ -0,0 +1,2 @@
+this is just for practice 
```

Comment:
why is this line added ? Is this necessary ?

Reply:
Yes it is needed